.ci/Jenkinsfile-extension-vsce

```diff
@@ -168,10 +175,21 @@ pipeline {
                     sh '''
                         # install the extension for its vsce dependency, required for building and publishing the extension
                         # and populate vscodeignore with exceptions to node_modules of the needed modules.
+                        # add the requirements.txt file to tie the extenstion to a server version
```

Comment:
Your comment [here](https://github.com/inmanta/vscode-inmanta/pull/1099#discussion_r1166518902) got me thinking about when we add this. It would be more in line with our current release process to add this in the "Setup next branch for major or minor release" stage so it is committed to the next (and after that the stable) branch.

Reply:
i'll have a look :p

Reply:
I made the change but I did not retest the whole flow as I expect it to behave in the same way but I would be harder for me to get the .vsix.